src/main/kotlin/com/example/mykku/notification/application/usecase/FcmNotificationSender.kt

```
@@ -0,0 +1,67 @@
+
+            if (response.failureCount > 0) {
+                response.responses.forEachIndexed { index, sendResponse ->
+                    if (!sendResponse.isSuccessful) {
+                        logger.error(
+                            "Failed to send FCM message to device: ${tokens[index].deviceId}, " +
+                            "error: ${sendResponse.exception?.message}"
+                        )
+                    }
+                }
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)